{
	"alias": "FEDORA-2024-b26f07d27b",
	"autokarma": true,
	"autotime": true,
	"bugs": [
		{
			"bug_id": 2282188,
			"title": "python-PyMySQL-1.1.1 is available",
			"bugzilla": {
				"bug_id": "2282188",
				"creation_ts": "2024-05-21 12:03:10 +0000",
				"short_desc": "python-PyMySQL-1.1.1 is available",
				"delta_ts": "2024-06-24 06:36:40 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "FutureFeature, Triaged",
				"priority": "unspecified",
				"bug_severity": "unspecified",
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "18024199",
						"comment_count": "0",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2024-05-21 12:03:10 +0000",
						"thetext": "Releases retrieved: 1.1.1\nUpstream release that is considered latest: 1.1.1\nCurrent version/release in rawhide: 1.1.0-6.fc40\nURL: https://pypi.python.org/pypi/PyMySQL/\n\nPlease consult the package updates policy before you issue an update to a stable branch: https://docs.fedoraproject.org/en-US/fesco/Updates_Policy/\n\n\nMore information about the service that created this bug can be found at: https://docs.fedoraproject.org/en-US/package-maintainers/Upstream_Release_Monitoring\n\n\nPlease keep in mind that with any upstream change, there may also be packaging changes that need to be made. Specifically, please remember that it is your responsibility to review the new version to ensure that the licensing is still correct and that no non-free or legally problematic items have been added upstream.\n\n\nBased on the information from Anitya: https://release-monitoring.org/project/7284/\n\n\nTo change the monitoring settings for the project, please visit:\nhttps://src.fedoraproject.org/rpms/python-PyMySQL"
					},
					{
						"isprivate": "0",
						"commentid": "18024200",
						"comment_count": "1",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2024-05-21 12:03:15 +0000",
						"thetext": "Scratch build failed. Details below:\n\nBuilderException: Build failed:\nCommand '['rpmbuild', '-D', '_sourcedir .', '-D', '_topdir .', '-bs', '/var/tmp/thn-htkjgzok/python-PyMySQL.spec']' returned non-zero exit status 1.\n\nStdOut:\nsetting SOURCE_DATE_EPOCH=1716249600\nerror: Bad file: ./PyMySQL-1.1.1.tar.gz: No such file or directory\n\nRPM build errors:\n    Bad file: ./PyMySQL-1.1.1.tar.gz: No such file or directory\n\n\nTraceback:\n  File \"/usr/local/lib/python3.12/site-packages/hotness/use_cases/package_scratch_build_use_case.py\", line 56, in build\n    result = self.builder.build(request.package, request.opts)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/local/lib/python3.12/site-packages/hotness/builders/koji.py\", line 229, in build\n    raise BuilderException(\n\nIf you think this issue is caused by some bug in the-new-hotness, please report it on the-new-hotness issue tracker: https://github.com/fedora-infra/the-new-hotness/issues"
					},
					{
						"isprivate": "0",
						"commentid": "18024201",
						"comment_count": "2",
						"who": {
							"text": "upstream-release-monitoring",
							"name": "Upstream Release Monitoring"
						},
						"bug_when": "2024-05-21 12:03:17 +0000",
						"thetext": "Created attachment 2034281\nUpdate to 1.1.1 (#2282188)"
					},
					{
						"isprivate": "0",
						"commentid": "18066561",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-15 15:53:25 +0000",
						"thetext": "FEDORA-2024-e7141ab284 (python-PyMySQL-1.1.1-1.fc39) has been submitted as an update to Fedora 39.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-e7141ab284"
					},
					{
						"isprivate": "0",
						"commentid": "18066562",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-15 15:53:26 +0000",
						"thetext": "FEDORA-2024-b26f07d27b (python-PyMySQL-1.1.1-1.fc40) has been submitted as an update to Fedora 40.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-b26f07d27b"
					},
					{
						"isprivate": "0",
						"commentid": "18066811",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-16 01:23:26 +0000",
						"thetext": "FEDORA-2024-b26f07d27b has been pushed to the Fedora 40 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-b26f07d27b`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-b26f07d27b\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18066833",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-16 01:40:27 +0000",
						"thetext": "FEDORA-2024-e7141ab284 has been pushed to the Fedora 39 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-e7141ab284`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-e7141ab284\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18076881",
						"comment_count": "7",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-24 01:38:49 +0000",
						"thetext": "FEDORA-2024-e7141ab284 (python-PyMySQL-1.1.1-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18076999",
						"comment_count": "8",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-24 06:36:40 +0000",
						"thetext": "FEDORA-2024-b26f07d27b (python-PyMySQL-1.1.1-1.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					}
				]
			}
		},
		{
			"bug_id": 2282821,
			"parent": true,
			"security": true,
			"title": "CVE-2024-36039 python-pymysql: SQL injection if used with untrusted JSON input",
			"bugzilla": {
				"bug_id": "2282821",
				"alias": "CVE-2024-36039",
				"creation_ts": "2024-05-23 05:06:44 +0000",
				"short_desc": "CVE-2024-36039 python-pymysql: SQL injection if used with untrusted JSON input",
				"delta_ts": "2024-06-17 18:59:23 +0000",
				"bug_status": "NEW",
				"keywords": "Security",
				"priority": "medium",
				"bug_severity": "medium",
				"depends_on": [
					"2282822",
					"2282823",
					"2292655"
				],
				"blocked": [
					{
						"bug_id": "2282826",
						"error": "NotPermitted"
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "18030667",
						"comment_count": "0",
						"who": {
							"text": "trathi",
							"name": "TEJ RATHI"
						},
						"bug_when": "2024-05-23 05:06:44 +0000",
						"thetext": "PyMySQL through 1.1.0 allows SQL injection if used with untrusted JSON input because keys are not escaped by escape_dict.\n\nReference:\nhttps://github.com/advisories/GHSA-v9hf-5j83-6xpp\n\nUpstream patch:\nhttps://github.com/PyMySQL/PyMySQL/commit/521e40050cb386a499f68f483fefd144c493053c"
					},
					{
						"isprivate": "0",
						"commentid": "18030670",
						"comment_count": "1",
						"who": {
							"text": "trathi",
							"name": "TEJ RATHI"
						},
						"bug_when": "2024-05-23 05:09:19 +0000",
						"thetext": "Created python-PyMySQL tracking bugs for this issue:\n\nAffects: fedora-all [bug 2282822]"
					},
					{
						"isprivate": "0",
						"commentid": "18069081",
						"comment_count": "10",
						"who": {
							"text": "gsuckevi",
							"name": "Guilherme de Almeida Suckevicz"
						},
						"bug_when": "2024-06-17 13:56:13 +0000",
						"thetext": "Created python-PyMySQL tracking bugs for this issue:\n\nAffects: epel-7 [bug 2292655]"
					}
				]
			}
		},
		{
			"bug_id": 2282822,
			"security": true,
			"title": "CVE-2024-36039 python-PyMySQL: SQL injection if used with untrusted JSON input [fedora-all]",
			"bugzilla": {
				"bug_id": "2282822",
				"creation_ts": "2024-05-23 05:09:08 +0000",
				"short_desc": "CVE-2024-36039 python-PyMySQL: SQL injection if used with untrusted JSON input [fedora-all]",
				"delta_ts": "2024-06-24 06:36:44 +0000",
				"bug_status": "CLOSED",
				"resolution": "ERRATA",
				"keywords": "Security, SecurityTracking",
				"priority": "medium",
				"bug_severity": "medium",
				"blocked": [
					{
						"bug_id": "2282821",
						"alias": "CVE-2024-36039",
						"creation_ts": "2024-05-23 05:06:44 +0000",
						"short_desc": "CVE-2024-36039 python-pymysql: SQL injection if used with untrusted JSON input",
						"delta_ts": "2024-06-17 18:59:23 +0000",
						"bug_status": "NEW",
						"keywords": "Security",
						"priority": "medium",
						"bug_severity": "medium",
						"depends_on": [
							"2282822",
							"2282823",
							"2292655"
						],
						"blocked": [
							{
								"bug_id": "2282826",
								"error": "NotPermitted"
							}
						],
						"long_desc": [
							{
								"isprivate": "0",
								"commentid": "18030667",
								"comment_count": "0",
								"who": {
									"text": "trathi",
									"name": "TEJ RATHI"
								},
								"bug_when": "2024-05-23 05:06:44 +0000",
								"thetext": "PyMySQL through 1.1.0 allows SQL injection if used with untrusted JSON input because keys are not escaped by escape_dict.\n\nReference:\nhttps://github.com/advisories/GHSA-v9hf-5j83-6xpp\n\nUpstream patch:\nhttps://github.com/PyMySQL/PyMySQL/commit/521e40050cb386a499f68f483fefd144c493053c"
							},
							{
								"isprivate": "0",
								"commentid": "18030670",
								"comment_count": "1",
								"who": {
									"text": "trathi",
									"name": "TEJ RATHI"
								},
								"bug_when": "2024-05-23 05:09:19 +0000",
								"thetext": "Created python-PyMySQL tracking bugs for this issue:\n\nAffects: fedora-all [bug 2282822]"
							},
							{
								"isprivate": "0",
								"commentid": "18069081",
								"comment_count": "10",
								"who": {
									"text": "gsuckevi",
									"name": "Guilherme de Almeida Suckevicz"
								},
								"bug_when": "2024-06-17 13:56:13 +0000",
								"thetext": "Created python-PyMySQL tracking bugs for this issue:\n\nAffects: epel-7 [bug 2292655]"
							}
						]
					}
				],
				"long_desc": [
					{
						"isprivate": "0",
						"commentid": "18030668",
						"comment_count": "0",
						"who": {
							"text": "trathi",
							"name": "TEJ RATHI"
						},
						"bug_when": "2024-05-23 05:09:08 +0000",
						"thetext": "\nMore information about this security flaw is available in the following bug:\n\nhttp://bugzilla.redhat.com/show_bug.cgi?id=2282821\n\nDisclaimer: Community trackers are created by Red Hat Product Security team on a best effort basis. Package maintainers are required to ascertain if the flaw indeed affects their package, before starting the update process."
					},
					{
						"isprivate": "0",
						"commentid": "18030669",
						"comment_count": "1",
						"who": {
							"text": "trathi",
							"name": "TEJ RATHI"
						},
						"bug_when": "2024-05-23 05:09:11 +0000",
						"thetext": "Use the following template to for the 'fedpkg update' request to submit an\nupdate for this issue as it contains the top-level parent bug(s) as well as\nthis tracking bug.  This will ensure that all associated bugs get updated\nwhen new packages are pushed to stable.\n\n=====\n\n# bugfix, security, enhancement, newpackage (required)\ntype=security\n\n# low, medium, high, urgent (required)\nseverity=medium\n\n# testing, stable\nrequest=testing\n\n# Bug numbers: 1234,9876\nbugs=2282821,2282822\n\n# Description of your update\nnotes=Security fix for [PUT CVEs HERE]\n\n# Enable request automation based on the stable/unstable karma thresholds\nautokarma=True\nstable_karma=3\nunstable_karma=-3\n\n# Automatically close bugs when this marked as stable\nclose_bugs=True\n\n# Suggest that users restart after update\nsuggest_reboot=False\n\n======\n\nAdditionally, you may opt to use the bodhi web interface to submit updates:\n\nhttps://bodhi.fedoraproject.org/updates/new"
					},
					{
						"isprivate": "0",
						"commentid": "18066563",
						"comment_count": "2",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-15 15:53:28 +0000",
						"thetext": "FEDORA-2024-e7141ab284 (python-PyMySQL-1.1.1-1.fc39) has been submitted as an update to Fedora 39.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-e7141ab284"
					},
					{
						"isprivate": "0",
						"commentid": "18066564",
						"comment_count": "3",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-15 15:53:32 +0000",
						"thetext": "FEDORA-2024-b26f07d27b (python-PyMySQL-1.1.1-1.fc40) has been submitted as an update to Fedora 40.\nhttps://bodhi.fedoraproject.org/updates/FEDORA-2024-b26f07d27b"
					},
					{
						"isprivate": "0",
						"commentid": "18066812",
						"comment_count": "4",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-16 01:23:28 +0000",
						"thetext": "FEDORA-2024-b26f07d27b has been pushed to the Fedora 40 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-b26f07d27b`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-b26f07d27b\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18066834",
						"comment_count": "5",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-16 01:40:29 +0000",
						"thetext": "FEDORA-2024-e7141ab284 has been pushed to the Fedora 39 testing repository.\nSoon you'll be able to install the update with the following command:\n`sudo dnf upgrade --enablerepo=updates-testing --refresh --advisory=FEDORA-2024-e7141ab284`\nYou can provide feedback for this update here: https://bodhi.fedoraproject.org/updates/FEDORA-2024-e7141ab284\n\nSee also https://fedoraproject.org/wiki/QA:Updates_Testing for more information on how to test updates."
					},
					{
						"isprivate": "0",
						"commentid": "18076883",
						"comment_count": "6",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-24 01:38:53 +0000",
						"thetext": "FEDORA-2024-e7141ab284 (python-PyMySQL-1.1.1-1.fc39) has been pushed to the Fedora 39 stable repository.\nIf problem still persists, please make note of it in this bug report."
					},
					{
						"isprivate": "0",
						"commentid": "18077000",
						"comment_count": "7",
						"who": {
							"text": "updates",
							"name": "Fedora Update System"
						},
						"bug_when": "2024-06-24 06:36:44 +0000",
						"thetext": "FEDORA-2024-b26f07d27b (python-PyMySQL-1.1.1-1.fc40) has been pushed to the Fedora 40 stable repository.\nIf problem still persists, please make note of it in this bug report."
					}
				]
			}
		}
	],
	"builds": [
		{
			"epoch": 0,
			"nvr": "python-PyMySQL-1.1.1-1.fc40",
			"release_id": 76,
			"signed": true,
			"type": "rpm",
			"package": {
				"noarch": [
					{
						"name": "python3-PyMySQL+ed25519",
						"version": "1.1.1",
						"release": "1.fc40",
						"arch": "noarch"
					},
					{
						"name": "python3-PyMySQL+rsa",
						"version": "1.1.1",
						"release": "1.fc40",
						"arch": "noarch"
					},
					{
						"name": "python3-PyMySQL",
						"version": "1.1.1",
						"release": "1.fc40",
						"arch": "noarch"
					}
				],
				"src": [
					{
						"name": "python-PyMySQL",
						"version": "1.1.1",
						"release": "1.fc40",
						"arch": "src"
					}
				]
			}
		}
	],
	"close_bugs": true,
	"comments": [
		{
			"id": 3562773,
			"text": "This update has been submitted for testing by jujens. ",
			"timestamp": "2024-06-15 15:53:23",
			"update_id": 618065,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3562774,
			"text": "This update's test gating status has been changed to 'ignored'.",
			"timestamp": "2024-06-15 15:53:23",
			"update_id": 618065,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3562991,
			"text": "This update has been pushed to testing.",
			"timestamp": "2024-06-16 01:23:42",
			"update_id": 618065,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3586454,
			"text": "This update has been submitted for stable by bodhi. ",
			"timestamp": "2024-06-23 01:23:42",
			"update_id": 618065,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		},
		{
			"id": 3587308,
			"text": "This update has been pushed to stable.",
			"timestamp": "2024-06-24 06:37:15",
			"update_id": 618065,
			"user": {
				"avatar": "https://apps.fedoraproject.org/img/icons/bodhi-24.png",
				"id": 91,
				"name": "bodhi",
				"openid": "bodhi.id.fedoraproject.org"
			},
			"user_id": 91
		}
	],
	"content_type": "rpm",
	"critpath_groups": "",
	"date_approved": "2024-06-23 01:23:41",
	"date_pushed": "2024-06-24 06:35:52",
	"date_stable": "2024-06-24 06:35:52",
	"date_submitted": "2024-06-15 15:53:23",
	"date_testing": "2024-06-16 01:22:40",
	"meets_testing_requirements": true,
	"notes": "Update to 1.1.1 to fix [CVE CVE-2024-36039](https://github.com/advisories/GHSA-v9hf-5j83-6xpp)",
	"pushed": true,
	"release": {
		"branch": "f40",
		"candidate_tag": "f40-updates-candidate",
		"composed_by_bodhi": true,
		"create_automatic_updates": false,
		"dist_tag": "f40",
		"eol": "2025-05-13",
		"id_prefix": "FEDORA",
		"long_name": "Fedora 40",
		"mail_template": "fedora_errata_template",
		"name": "F40",
		"override_tag": "f40-override",
		"package_manager": "dnf",
		"pending_signing_tag": "f40-signing-pending",
		"pending_stable_tag": "f40-updates-pending",
		"pending_testing_tag": "f40-updates-testing-pending",
		"stable_tag": "f40-updates",
		"state": "current",
		"testing_repository": "updates-testing",
		"testing_tag": "f40-updates-testing",
		"version": "40"
	},
	"require_bugs": true,
	"require_testcases": true,
	"severity": "medium",
	"stable_days": 7,
	"stable_karma": 3,
	"status": "stable",
	"suggest": "unspecified",
	"test_gating_status": "ignored",
	"title": "python-PyMySQL-1.1.1-1.fc40",
	"type": "security",
	"url": "https://bodhi.fedoraproject.org/updates/FEDORA-2024-b26f07d27b",
	"unstable_karma": -3,
	"updateid": "FEDORA-2024-b26f07d27b",
	"user": {
		"avatar": "https://seccdn.libravatar.org/avatar/189dd5d70e090954ff4dd523d5e4477fb61b981e88b5341558b6e032cec7d3ad?s=24&d=retro",
		"email": "jujens@jujens.eu",
		"groups": [
			{
				"name": "packager"
			},
			{
				"name": "ipausers"
			},
			{
				"name": "fedora-contributor"
			},
			{
				"name": "signed_fpca"
			},
			{
				"name": "fedorabugs"
			}
		],
		"id": 260,
		"name": "jujens",
		"openid": "jujens.id.fedoraproject.org"
	},
	"version_hash": "ab8762f75277a571c1b04bad1b534b4eb15d29bd"
}
